apps/src/templates/sectionProgress/progressTables/ProgressTableSummaryView.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  progressForLesson,
+  lessonIsAllAssessment
+} from '@cdo/apps/templates/progress/progressHelpers';
+import {scriptDataPropType} from '../sectionProgressConstants';
+import {
+  getCurrentScriptData,
+  jumpToLessonDetails
+} from '@cdo/apps/templates/sectionProgress/sectionProgressRedux';
+import ProgressTableContainer from './ProgressTableContainer';
+import ProgressTableSummaryCell from './ProgressTableSummaryCell';
+import progressTableStyles from './progressTableStyles.scss';
+import SummaryViewLegend from '@cdo/apps/templates/sectionProgress/summary/SummaryViewLegend';
+
+const MIN_COLUMN_WIDTH = 40;
+class ProgressTableSummaryView extends React.Component {
```

Comment:
(minor) A header comment here would be useful. Things that might be useful to touch upon:
- This component displays a bar that summarizes the progress for each lesson in a single script.
- It glues together the components specific to summary view (ProgressTableSummaryCell) with the shared progress view components (ProgressTableContainer)
- There is also a sister component ProgressTableDetailView that presents a more detailed, level-by-level view of the same data.

Reply:
I should have seen this coming! Will update.